### PR TITLE
refactor: delete all pn-landing reference

### PR DESCRIPTION
**Pull request by leleOFA**
## Short description
all pn-landing reference

### CODE REVIEW

**Review by AndreaCimini90 — CHANGES_REQUESTED** [review]
- Check cra_buildspec.yaml row 19
- Check README row 17, 48
- Check one-cdn.yaml
- Check env-cdn.sh

File: cra_buildspec.yaml [hunk 1]
```diff
@@ -15,10 +15,6 @@ phases:
       - echo launching yarn install in $SUB_PRJ_NAME...
       - cd $SUB_PRJ_NAME
       - yarn install
-      - | 
-        if ( [ "$SUB_PRJ_NAME" != "packages/pn-landing-webapp" ] ) then 
-          echo Running tests.. && yarn test
```
Comment (AndreaCimini90) [inline cra_buildspec.yaml:20]: the condition says that if $SUB_PRJ_NAME **isn't** packages/pn-landing-webapp, tests must be run. so adjust this condition

File: aws-cdn-templates/one-cdn.yaml [hunk 2]
```diff
@@ -63,16 +63,10 @@ Parameters:
 
   S3LogsBucket:
     Type: String
-    Default: '-'
+    Default: "-"
     Description: Access Log bucket name
 
 Conditions:
-  # Is landing site if name startsWith 'web-landing-cdn-'
-  IsLanding:
```
Comment (AndreaCimini90) [inline aws-cdn-templates/one-cdn.yaml:71]: IsLanding is used around the file. Adjust where it is used

**Review by AndreaCimini90 — CHANGES_REQUESTED** [review]

File: cra_buildspec.yaml (hunk 1 above)
Comment (leleOFA) [reply cra_buildspec.yaml:20]: done

File: aws-cdn-templates/one-cdn.yaml (hunk 2 above)
Comment (leleOFA) [reply aws-cdn-templates/one-cdn.yaml:71]: done

**Review by AndreaCimini90 — APPROVED** [review]

**Review by AndreaCimini90 — APPROVED** [review]

**Pull request merged**